apps/polls/views.py

```diff
@@ -55,8 +35,18 @@ def get_success_url(self):
             'dashboard-project-list',
             kwargs={'organisation_slug': self.organisation.slug, })
 
+    def get_or_create_poll(self):
```

Comment:
Why not use https://docs.djangoproject.com/en/1.11/ref/models/querysets/#get-or-create ?